modules/developer-cli-odo-creating-a-java-application-by-using-a-devfile-in-a-disconnected-cluster.adoc

```diff
@@ -0,0 +1,193 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/
+
+[id="developer-cli-odo-creating-a-java-application-by-using-a-devfile-in-a-disconnected-cluster_{context}"]
+
+= Creating a Java application by using a devfile in a disconnected cluster
+
+[WARNING]
+====
+This procedure is using external dependencies such as `quay.io/eclipse/che-java11-maven:nightly` or an example application `springboot-ex` that are not maintained by Red Hat. These dependencies are not maintained with the documentation and their functionality cannot be guaranteed. 
```

Comment:
```suggestion
This procedure uses external dependencies, such as `quay.io/eclipse/che-java11-maven:nightly` or an example application `springboot-ex`, that are not maintained by Red Hat and whose functionality may change. 
```
- s/is using/uses/ because the present progressive, "is using", suggests a temporary activity.
- We should not imply that the functionality of maintained software is guaranteed. Doing so has legal implications, which is outside the scope of our product documentation.